Maven.gitignore

```diff
@@ -7,3 +7,6 @@ release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
 .mvn/timing.properties
+
+# Exclude maven wrapper
+!/.mvn/wrapper/maven-wrapper.jar
```

Comment:
> Is Maven a tool rather than a programming language? If so, it should live under the Global namespace...

Yes. Maven is a build and dependency management tool for the JVM.

> Is it common for people to use Java.gitignore without Maven.gitignore?

I would not say it is common but there are of course situations where one would not want to use Maven. Although I think Maven is still a very popular build/dependency management tool for Java projects.

> Should all the Maven rules from Java.gitignore live in Maven.gitignore?

No, because of the above reason.